src/game/hud.gd

```diff
@@ -36,3 +46,57 @@ func addAbility(buttontext: String, abilityResource: Ability) -> void:
 func clearAbilities() -> void:
 	for element in abilityBox.get_children():
 		element.queue_free()
+
+func addItemToPickUp(itemRes: ItemResource) -> void:
+	var newItemIcon: Control = itemPickUpScene.instance()
+	newItemIcon.setItemResource(itemRes)
+	if Characters.getMyCharacterResource().canPickUpItem(itemRes):
+		itemIntBox.add_child(newItemIcon)
+	interactable.append(itemRes)
+	interactUi.append(newItemIcon)
+
+func removePickUpItem(itemRes: ItemResource) -> void:
+	var index: int
+	index = interactable.find(itemRes)
+	interactUi[index].queue_free()
+	interactUi.pop_at(index)
+	interactable.pop_at(index)
+
+func hidePickUpButtons() -> void:
+	for interactIconInstance in interactUi:
+		itemIntBox.remove_child(interactIconInstance)
+
+func refreshPickUpButtons() -> void:
+	for buttonInstance in interactUi:
```

Comment:
>interactUi
I get that this is technically following convention, but Ui as opposed to UI always just doesn't look right to me lol